root/etc/e-smith/events/actions/nethserver-dc-user-modify

```diff
@@ -49,25 +49,33 @@ if ($? != 0) {
 
 #Create a temporary ldif file (root:root 0600)
 my $fileName = File::Temp->new(
-    UNLINK   => 0,
+    UNLINK   => 1,
     TEMPLATE => "$userName.XXXXXXXXXX",
     DIR => '/var/lib/machines/nsdc/var/spool/'
 );
 
-#create the ldif content
+#we need the relative path in the container
+my $relativePathFile = $fileName; 
+$relativePathFile =~ s|/var/lib/machines/nsdc||;
```

Comment:
As alternative to regexp substitution, declare a variable `$nsprefixpath="/var/lib/machines/nsdc"` and prepend it to `DIR` above